torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -301,6 +301,21 @@ AOTITorchError aoti_torch_mm_out(
   });
 }
 
+// Function to check existence of inf and NaN
+AOTITorchError aoti_check_inf_and_nan(AtenTensorHandle tensor) {
+  AOTI_TORCH_CONVERT_EXCEPTION_TO_ERROR_CODE({
+    at::Tensor* check_tensor = tensor_handle_to_tensor_pointer(tensor);
+    auto flattened = check_tensor->view({-1});
+
+    for (int64_t i = 0; i < flattened.numel(); i++) {
+      auto value = flattened[i].item<float>();
+      if (std::isinf(value) || std::isnan(value)) {
+        assert(false);
```

Comment:
Are we ok with using `assert` here? It will be wiped out without `NDEBUG`.

Reply:
I think it should be fine. We will not have this in prod or for testing etc. This, in theory, would only be plugged in when we are debugging.